packages/typespec-ts/test/integration/modelInheritance.spec.ts

```diff
@@ -14,7 +15,7 @@ describe("ModelsInheritance Rest Client", () => {
     });
   });
 
-  const validBody = { name: "abc", age: 32, smart: true };
+  const validBody: Siamese = { name: "abc", age: 32, smart: true };
```

Comment:
not sure if we shoud add this, as we are recommending our customers to not use type annotation now.

Reply:
I think this is only for inheritance test case, we could add the interface name explicitly for type checking.